.CI/create_feedstocks.py

```diff
@@ -205,6 +173,7 @@ def write_token(name, token):
                     # Capture the output, as it may contain the GH_TOKEN.
                     out = subprocess.check_output(['git', 'push', 'upstream_with_token', 'HEAD:master'], cwd=feedstock_dir,
                                                   stderr=subprocess.STDOUT)
+                    subprocess.check_call(['conda', 'smithy', 'register-github', '--add-teams', feedstock_dir] + owner_info)
```

Comment:
Tries adding the feedstock's team and members to it only after pushing to the feedstock has succeeded. This way permissions are only given once the feedstock is fully up and operational. If conversion fails before this point and the feedstock is malformed, the maintainers won't have permissions yet. This matches with how team member additions are handled now.

Reply:
FYI, with the webservice in mind, the maintainers will have permission when the interpreter reaches here unless the webservice failed.

Reply:
That's correct.

Reply:
IOW this is only a safeguard that we will probably drop down the line once we are satisfied with the webservice's behavior.

Reply:
I guess we can drop this line now. That said, am ok with leaving it as a safeguard.